test/e2e/utils/test_context.go

```diff
@@ -70,9 +70,9 @@ func NewTestContext(binaryName string, env ...string) (*TestContext, error) {
 		Domain:     "example.com" + testSuffix,
 		Group:      testGroup,
 		Version:    "v1alpha1",
-		Kind:       "Foo" + testSuffix,
-		Resources:  "foo" + testSuffix + "s",
-		ImageName:  "e2e-test/controller-manager:" + testSuffix,
+		Kind:       fmt.Sprintf("Foo%s", testSuffix),
+		Resources:  fmt.Sprintf("foo%s", testSuffix) + "s",
+		ImageName:  fmt.Sprintf("e2e-test/controller-manager:%s", testSuffix),
```

Comment:
Can you revert this change? It has nothing to do with this PR and is only done partially.